mmdet/datasets/pipelines/transforms.py

```diff
@@ -2079,6 +2079,11 @@ def _mosaic_transform(self, results):
             mosaic_bboxes[:, 1::2] = np.clip(mosaic_bboxes[:, 1::2], 0,
                                              2 * self.img_scale[0])
             mosaic_labels = np.concatenate(mosaic_labels, 0)
+            
+            mosaic_filter = np.prod(mosaic_bboxes[:, 2:4] - \
```

Comment:
If you want to add filtering rules, it is recommended to create a new internal method, and then put the hyperparameters in the initialization method.

```python
def __init__(self,
                 img_scale=(640, 640),
                 center_ratio_range=(0.5, 1.5),
                 min_bbox_size,
                 pad_val=114):
        self.min_bbox_size=min_bbox_size


    def _filter_box_candidates(self, bbox):

        ....
        return ((w > self.min_bbox_size)
                & (h > self.min_bbox_size)
              
```

Reply:
Thank you for your kindly reminding and advice. I have created a new internal method "_filter_box_candidates" and put the hyperparameters "min_bbox_size" in the initialization method. 
(The function Shear._filter_invalid at <mmdet/datasets/pipelines/auto_augment.py> are reference to the created internal method "_filter_box_candidates".)